QualityUnit/liveagent-theme
# [494] feat: pricing shortcode redesign

**Changes proposed in this Pull Request**
Redesigned pricing element, merged both designes into one, added tags. Shortcode now takes only price and title.

**Testing instructions**
Go to e.g. [/ticketing-software/](http://liveagent.local:3000/ticketing-software/), scroll a bit and check if everything looks as in https://www.figma.com/file/7cBp0wE3ZdgsYoGHH6SIzj/LA-WEB-RfD---pricing-element?node-id=5%3A2

Elementor element was changed to structure: 
<img width="229" alt="Screenshot 2022-07-11 at 12 22 00" src="https://user-images.githubusercontent.com/22455555/178244035-c5c25d1d-e899-4769-9798-a4b5602c81b2.png">

main section has classname: "BlockPricing"
column has: "wrapper wrapper__wide"
and button on the bottom has added class: "compare-plans"

**Checklist**

Close QualityUnit/web-issues#494
